awscli/customizations/wizard/wizards/iam/new-role-wip.yml

```diff
@@ -233,3 +233,17 @@ execute:
         # This role_arn comes from the `output_var` in the previous
         # executor step.
         role_arn: "{role_arn}"
+__OUTPUT__:
+  value: |
+    Wizard has created a role:
```

Comment:
For this first line, let's update it to `Wizard successfully created role with following properties:` to make it a little more explicit about what we are listing.